ext/build_defaults.yaml

```diff
@@ -26,9 +26,9 @@ build_msi:
     repo: 'git://github.com/puppetlabs/hiera.git'
   sys:
     ref:
-      x86: '8db9d84da9950760144b5dfcd807213eecee4842'
-      x64: '12030f11e9bb2f085c68108bff34be6956b25df9'
-    repo: 'git://github.com/puppetlabs/puppet-win32-ruby.git'
+      x86: 'debafa740bca3f799151a3f59f7c028ac07a9bd3'
+      x64: '17e455a5be2743774c4357ee24770fe411ffb837'
+    repo: 'git://github.com/Iristyle/puppet-win32-ruby.git'
```

Comment:
@Iristyle ready to be updated:

1.9.3-x86: [2f25839c5c11ec10b781debf4808095d6895a844](
https://github.com/puppetlabs/puppet-win32-ruby/commit/2f25839c5c11ec10b781debf4808095d6895a844)
2.0.0-x64: [017c63ca11e38f74b7e011ecd2006100452c620b](https://github.com/puppetlabs/puppet-win32-ruby/commit/017c63ca11e38f74b7e011ecd2006100452c620b)

Also need to point back to `puppetlabs` repo.